### DISCUSSION TITLE
プロファイル作成時の通知をオン/オフにしたいです - 思惟奈ちゃんヘルプセンター

### DISCUSSION THREAD

**Discussion by giscus** [SinaKitagami/sina-helpcenter#7] [チャット]
# プロファイル作成時の通知をオン/オフにしたいです - 思惟奈ちゃんヘルプセンター

Discordボット 思惟奈ちゃんのヘルプセンターです。何か不具合が発生した場合や、お困りの際にご利用ください。

https://help.sina-chan.com/category/%E8%A8%AD%E5%AE%9A/%E3%83%97%E3%83%AD%E3%83%95%E3%82%A1%E3%82%A4%E3%83%AB%E4%BD%9C%E6%88%90%E6%99%82%E3%81%AE%E9%80%9A%E7%9F%A5%E3%82%92%E3%82%AA%E3%83%B3-%E3%82%AA%E3%83%95%E3%81%AB%E3%81%97%E3%81%9F%E3%81%84%E3%81%A7%E3%81%99

**Comment by T-taku** [comment]
何かわからないことがあれば、ここのコメントにお書きください!
(GitHubのアカウントが必要です。)
